example/settings.gradle.kts

```diff
@@ -45,6 +45,12 @@ fun String.includeProject(id: String, path: String = "$this/$id") {
   includeProject("tiger.eclipse", "tiger/manual/tiger.eclipse")
   includeProject("tiger.intellij", "tiger/manual/tiger.intellij")
 }
+"tiger/spoofax3".run {
+  includeProject("tiger", "tiger/spoofax3/tiger")
```

Comment:
This project has the same name as the other Tiger project, which causes problems. Can you rename this to `tiger.spoofax3`, as https://github.com/metaborg/spoofax-pie/issues/69 is not an issue on Gradle 6.

Reply:
:+1: Okay, renamed.